presto-docs/src/main/sphinx/release/release-303.rst

```diff
@@ -0,0 +1,53 @@
+===========
+Release 303
+===========
+
+General Changes
+---------------
+
+* Fix incorrect padding for ``CHAR`` values containing Unicode supplementary characters. Previously, such
+  values would be incorrectly padded with too few spaces. (:issue:`195`)
+* Fix an issue where a union of a table with a values statement would execute on a single node,
+  which could lead to out of memory errors. (:issue:`207`)
+* Fix ``/v1/info`` to report started status after all plugins have been registered and
+  initialized. (:issue:`213`)
+* Improve performance of window functions by avoiding unnecessary data exchanges over
+  the network. (:issue:`177`)
+* Add support for choosing distribution type for semi join based on cost when
+  ``join_distribution_type='AUTOMATIC'``. (:issue:`160`)
+* Expand grouped execution support to window functions making it possible
+  to execute them with less peak memory usage. (:issue:`169`)
+
+CLI Changes
+-----------
+
+* Add ``--progress`` option to show query progress in batch mode. (:issue:`34`)
+
+Hive Connector Changes
+----------------------
+
+* Fix failure when reading DWRF flat map encoded columns. (:issue:`158`)
```

Comment:
"query failure"?

Reply:
We can remove this since no one uses this feature. The release notes didn't mention when flat map was added.